### ISSUE TITLE
[Research] OUI Compliance audit of the `vis_default_editor` plugin

### ISSUE THREAD

**Issue by BSFishy** [opensearch-project/OpenSearch-Dashboards#4096]
Tracking issue for OUI compliance audit of the `vis_default_editor` plugin.

Follow the steps in #3945 to complete the audit, then open another issue with findings. Once the work of performing the audit is done, this issue can be closed.

If this issue isn't already assigned to someone and you'd like to take on this work, please ping @BSFishy in the comments and I can assign it to you.

**Comment by joshuarrrr** [comment]
The styling

https://github.com/opensearch-project/OpenSearch-Dashboards/blob/fe1f1d18ea9b26ddb19ae805f676b6e09798b294/src/plugins/vis_default_editor/public/_sidebar.scss#L29-L35

affects a lot of visualizations (see https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4390)
